README-Copy1.md

```diff
@@ -0,0 +1,41 @@
+# qpub - q(uick) publishing of python projects
```

Comment:
I love this name. 

I think there is still an overlap of tools/scope. This is part config, part test suite/build tool unification. It's fine, but I keep getting tripped up on that. 

And now that I'm thinking about it, if you take the scope at the higher level of "publishing" (like you do) then it does make more sense.